app/src/main/java/com/example/arc_exapmle/note/NoteDao.kt

```diff
@@ -7,22 +7,22 @@ import androidx.room.*
  interface NoteDao {
 
     @Insert
-    fun insert(note: NoteEntity)
+    suspend fun insert(note: NoteEntity)
 
     @Update
-    fun update(note: NoteEntity)
+    suspend fun update(note: NoteEntity)
 
     @Delete
-    fun delete(note: NoteEntity)
+    suspend fun delete(note: NoteEntity)
 
     @Query("DELETE FROM note_table")
-    fun deleteAll()
+    suspend fun deleteAll()
 
     @Query("SELECT * FROM note_table WHERE user = :userID")
-     fun getAllNotes(userID: Int): LiveData<List<NoteEntity>>
+    fun getAllNotes(userID: Int): LiveData<List<NoteEntity>>
 
    @Query("SELECT * FROM note_table")
-   fun getEveryNoteThereIs(): LiveData<List<NoteEntity>>
+    fun getEveryNoteThereIs(): LiveData<List<NoteEntity>>
```

Comment:
behtar nist hala ke darim az coroutines estefade mikonim inaro ham az LiveData be coroutines tabdil konim?